ipasetup.py.in

```diff
@@ -19,20 +19,21 @@ import sys
 
 
 PACKAGE_VERSION = {
-    'cryptography': 'cryptography >= 0.9',
+    'cryptography': 'cryptography >= 1.5',
     'dnspython': 'dnspython >= 1.11.1',
-    'gssapi': 'gssapi > 1.1.2',
+    'gssapi': 'gssapi >= 1.2.0',
     'ipaclient': 'ipaclient == @VERSION@',
     'ipalib': 'ipalib == @VERSION@',
     'ipaplatform': 'ipaplatform == @VERSION@',
     'ipapython': 'ipapython == @VERSION@',
     'ipaserver': 'ipaserver == @VERSION@',
     'kdcproxy': 'kdcproxy >= 0.3',
     'netifaces': 'netifaces >= 0.10.4',
-    'python-nss': 'python-nss >= 0.16',
     'pyldap': 'pyldap >= 2.4.15',
+    'pytest': 'pytest < 3.0',
```

Comment:
Why pytest < 3 ?  in specfile we have >=2.6.  If pytest > 3 does not work with IPA, a ticket should be opened and specfile fixed as well.